src/js/components/Select/__tests__/Select-test.js

```diff
@@ -201,6 +201,27 @@ describe('Select', () => {
     expect(window.scrollTo).toBeCalled();
   });
 
+  test('search with search value prop', () => {
+    jest.useFakeTimers();
+    const onSearch = jest.fn();
+    const { container } = render(
+      <Select
+        id="test-select"
+        options={['one', 'two']}
+        onSearch={onSearch}
+        searchValue="two"
+      />,
+    );
+    expect(container.firstChild).toMatchSnapshot();
+
+    // advance timers so select can open
+    act(() => {
+      jest.advanceTimersByTime(200);
+    });
+
+    expect(document.activeElement).toMatchSnapshot();
+  });
+
```

Comment:
Hi Gabriel, that test isnt very effective, I recommend this approach:

```js
  test('search with search value prop', () => {
    const onSearch = jest.fn();
    const { getByPlaceholderText,getByTestId } = render(
      <Select
        id="test-select"
        options={['one', 'two']}
        onSearch={onSearch}
        searchValue="two"
        searchPlaceholder="search-placeholder"
        data-testid="test"
      />,
    );
    fireEvent.click(getByTestId('test'));
    const searchInput = getByPlaceholderText('search-placeholder');
    expect(searchInput.value).toBe("two");
  });
  ```

Reply:
Nice! Thanks a lot @negreirosleo.